assets/src/components/form/base/with-form-container-and-placeholder.js

```diff
@@ -18,27 +18,24 @@ export default compose( [
 	withInstanceId,
 	createHigherOrderComponent(
```

Comment:
this likely doesn't even need to be a HOC as it doesn't modify the child form in any way.
could likely just use the `children` prop and `React.Children.only(children)` to enforce a single form.
that said, we don't need to do that as part of this PR, just commenting for the sake of commenting